### ISSUE TITLE
incorrect handling of the `@container` at-rule

### ISSUE THREAD

**Issue by phiw13** [clean-css/clean-css-cli#82]
The (first) selector inside the at-rule together with the opening `{`  of the rule-block and the first property-value pair is removed. (treated as an invalid property name).

input:

```
@container item (min-width: 400px) {
  .item_inner {
    display: flex;
    flex-direction: row;
    align-items: center;
  }
}
```

output: 

```
@container item (min-width: 400px){flex-direction:row;align-items:center}
```

`.item_inner { display: flex;`  is removed, treated as an invalid peoperty name

Testedon macOS 12 with clean-css-cli 5.6.1  and reproduced  on the clean-css website.

**Comment by phiw13** [comment]
BTW - the new `@layer` at-rule suffers from the same issue.

Basic example(s) on the [MDN page](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer).

**Comment by jakubpawlowicz** [comment]
Both are fixed in clean-css-cli 5.6.2.
